tgui/src/components/display.ract

```diff
@@ -1,7 +1,7 @@
 <div class='display'>
   {{#if title}}
     <header>
-      <h3>{{title}}</h3>
+      <h3>{{{title}}}</h3>
```

Comment:
Otherwise it won't parse the HTML tags in title, which I use in the identity manager